modules/imap/handler_modules.php

```diff
@@ -868,6 +882,72 @@ public function process() {
             $this->out('do_not_flag_as_read_on_open', $this->user_config->get('unread_on_open_setting', false));
         }
     }
+    public function validationMessage($email, $subject, $msg, $imap, $cache, $imap_server_id) {          
+        // 1. Check Suspicious Phrases or Requests
```

Comment:
Phrase in French is a false friend to English phrase:
https://en.wiktionary.org/wiki/phrase

"phrase" in French translates "sentence" in English.

Since we may not be checking full sentences, you may want to use "suspicious terms"

Reply:
Exactly